tests/main/snap-confine-from-core/task.yaml

```diff
@@ -19,8 +19,7 @@ execute: |
     fi
 
     echo "Ensure we re-exec by default"
-    snap list
-    journalctl | MATCH "DEBUG: restarting into"
+    /usr/bin/env SNAPD_DEBUG=1 snap list 2>&1 | MATCH "DEBUG: restarting into"
```

Comment:
Can you please drop the env and the list command (you can just say `SNAPD_DEBUG=1 snap`) and add a comment explaining what is going on there.